packages/extension-sample/src/setup-app.ts

```diff
@@ -9,12 +9,12 @@ import React from 'react'
 import loadable from '@loadable/component'
 import {ApplicationExtension, IRouteConfig} from '@salesforce/pwa-kit-react-sdk/ssr/universal/extensibility'
 
-import withRedBorder from './components/with-red-border'
+import withRedBorder from '*/components/with-red-border'
 import {ReactExtensionConfig as Config} from './types'
 
-const SamplePage = loadable(() => import('./pages/sample'))
-const defaultPath: string = '/sample-page'
+const SamplePage = loadable(() => import('*/pages/sample'))
```

Comment:
Currently, this wildcard import means that it'll resolve to the sample file in `extension-sample` package. However, if the _base project_ has the same exact file, then it'll resolve to that one instead 🤔 Hmm, is this ok?

I guess in my mind, I've assumed the wildcard import to mean the following heuristic:
1.  Find the file locally (in the package where the import statement is)
2.  If not found, then look elsewhere, starting from the base project